src/App.js

```diff
@@ -1,29 +1,35 @@
 import './App.css';
 import { Component } from 'react';
 import SearchForm from './Components/SearchForm';
+import ResultsDisplay from './Components/ResultsDisplay';
 import axios from 'axios';
 import Card from 'react-bootstrap/Card';
 
+
 class App extends Component {
   constructor(props) {
     super(props);
 
     this.state = {
       location: {},
       error: '',
-      map: null,
     };
   }
 
   getLocation = async (q) => {
     let API = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_KEY}&q=${q}&format=json`;
+    let errorFound = false;
 
     try {
       const res = await axios.get(API);
       this.setState({location: res.data[0]});
     } catch(err) {
       this.setState({error: 'Status Code ' + err.response.status + ': ' + err.response.data.error
                      + '. Please Modify your query and try again.'});
+      errorFound = true;
+    }
+    if(!errorFound) {
+      this.setState({error: ''});
```

Comment:
This logic is great! A great additional step to ensure the state is set back to false for the error message after it has been used.